src/classes/Queue.js

```diff
@@ -0,0 +1,17 @@
+import { DoubleLinkedList } from './DoubleLinkedList';
+
+class Queue {
+  constructor() {
+    this._list = new DoubleLinkedList();
+  }
+
+  add(data) {
+    this._list.push(data);
+  }
+
+  pop() {
+    return this._list.shift();
+  }
```

Comment:
I'm concerned about the fact that this is returning a node. I think it would be more intuitive if we returned `node.data`, because the queue is supposed to handle data

```suggestion
  pop() {
    const node = this._list.shift();
    return node.data;
  }
```